web/components/layout-footer.tsx

```diff
@@ -14,16 +14,6 @@ export const Footer = () => {
           >
             Granola
           </a>
-          . The source code is available on{' '}
```

Comment:
The deletion of the period at the beginning of this line was a mistake.

Reply:
@robinbb addressed in new commit below.